docs/moosedocs.py

```diff
@@ -4,7 +4,7 @@
 import pdb
 
 # Locate MOOSE directory
-MOOSE_DIR = os.getenv('MOOSE_DIR', os.path.join(os.getcwd(), '..', 'moose'))
+MOOSE_DIR = os.getenv('MOOSE_DIR', os.path.join(os.getcwd(), '..', '..', 'moose'))
```

Comment:
You shouldn't be changing this, if this is not working for you we need to improve the logic so that it does.